org.eclipse.jdt.ui/core refactoring/org/eclipse/jdt/internal/corext/refactoring/code/IntroduceParameterRefactoring.java

```diff
@@ -489,7 +489,7 @@ private List<String> guessTempNamesFromMethodInvocation(MethodInvocation selecte
 			if (! Character.isUpperCase(firstAfterPrefix))
```

Comment:
Doesn't this need to be changed as well?

Reply:
Yes, I think so - and there is more code to change. I just wanted to have a small one to get familar with the contribution process in jdt.ui now. Maybe that one is too much..

Reply:
Should we ever try to adopt full java spec in jdt we can investigate again. To shorten the queue (summer holiday ended🤷‍♂️) I close this one.